.gitlab-ci.yml

```diff
@@ -48,67 +48,75 @@ generate_code:
 
 build_eds_image_amd64:
   stage: image
-  tags: ["runner:docker", "size:large"]
+  tags: 
+    - "arch:amd64"
```

Comment:
checking if we ought to use `size:large` tags for the new runners as well

Reply:
from inspecting our runner nodes i don't see any indication of a `size` tag or label

Reply:
from inspecting our runner nodes i don't see any indication of a `size` tag or label